roles/coachview/vars/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+coachview_cds_url: https://172.29.1.207/api/contests/
```

Comment:
I think we should put this file in the .gitignore as we already have a defaults which documents the variables.

Reply:
git rm'd it and the iptablesrules one.

Reply:
can we put a roles/*/var/main.yml in the .gitignore?